<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_LookUp</class>
 <widget class="QDialog" name="Dialog_LookUp">
  <property name="windowTitle">
   <string>Edit Recipe</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>309</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>12</number>
   </property>
   <item row="1" column="1">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_1">
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="add_PB">
       <property name="text">
        <string>...</string>
       </property>
       <property name="shortcut">
        <string>Alt+I</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/resources/plus.png</normaloff>
        </iconset>
       </property>
       <property name="toolTip">
        <string>Insert new Ingredient</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="delete_PB">
       <property name="text">
        <string>...</string>
       </property>
       <property name="shortcut">
        <string>Alt+D</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/resources/minus.png</normaloff>
        </iconset>
       </property>
       <property name="toolTip">
        <string>Delete Current Ingredient</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="up_PB">
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/resources/up-16.png</normaloff>
        </iconset>
       </property>
       <property name="toolTip">
        <string>Move Ingredient Up </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="down_PB">
       <property name="text">
        <string>...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/resources/down-16.png</normaloff>
        </iconset>
       </property>
       <property name="toolTip">
        <string>Move Ingredient Down</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>15</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="files_PB">
       <property name="text">
        <string>Select Files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="folders_PB">
       <property name="text">
        <string>Select Folders</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>15</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save_PB">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="shortcut">
        <string>Alt+S</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>7</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_PB">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_4">
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>add_PB</tabstop>
  <tabstop>delete_PB</tabstop>
  <tabstop>up_PB</tabstop>
  <tabstop>down_PB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
